requirements.txt

```diff
@@ -2,4 +2,5 @@ uvicorn
 httpx
 fastapi
 typing
-gitpython
\ No newline at end of file
+gitpython
+requests
```

Comment:
In this case since the codebase uses `httpx`, we should use `httpx` (as in core repo we are using `requests` and I asked you to use `requests`).
Just align yourself with the repo style ;)

Or there is a specific reason for installing and using `requests`?